lua/lspconfig/server_configurations/hdl_checker.lua

```diff
@@ -0,0 +1,23 @@
+local configs = require 'lspconfig.configs'
+local util = require 'lspconfig.util'
+
+local server_name = 'hdl_checker'
+
+configs[server_name] = {
```

Comment:
this isn't how you define a server, you need to return this table, there is no "configs" anymore. See any other configuration.

Reply:
Ok I followed the example in the README
Will try again